### ISSUE TITLE
Rest parameters

### ISSUE THREAD

**Issue by drcmda** [pedronauck/docz#307]
## Bug Report

Rest params crash docz

**To Reproduce**

```jsx
<Playground>
  <RenderProps>
    {({ something, ...props }) => <div style={props}>{something}</div>}
  </RenderProps>
</Playground>
```

Babel shouldn't have any problems with it, and the strangest part is that if rest props are edited in live with the playground component, then it works, but if it compiles from code it fails.

```
index.js?1d39:1 Uncaught TypeError: Cannot read property 'toString' of undefined
    at JSON.parse.e (index.js?1d39:1)
    at _o (index.js?1d39:1)
    at new Po (index.js?1d39:1)

react-dom.development.js?61bb:14389 The above error occurred in the <Po> component:
    in Po (created by BasePlayground)
    in BasePlayground (created by Context.Consumer)
    in Component (at Spring.mdx:12)
    in div (created by MDXTag)
    in MDXTag (created by Context.Consumer)
    in Component (at Spring.mdx:7)
    in Component (created by o)
    in o (created by AsyncComponent)
    in AsyncComponent (created by t)
    in t (created by Context.Consumer)
    in b (created by s)
    in s (created by Context.Consumer)
    in Component (created by AsyncRoute)
```

**Comment by joshwcomeau** [comment]
I'm seeing this as well, when I try using static property initializers. I had the idea to create a state-manager component within a Playground. 

Here's a minimum reproducible example:

```js
## Toggleable

<Playground>
  {() => {
    class StateManager extends Component {
      state = { toggled: false }; // This fella's the problem
      render() {
        return <div>Hello World</div>;
      }
    }
  }}
</Playground>
```

Really enjoying Docz, great project. We're considering switching from Storybook on Guppy :D

**Comment by pedronauck** [comment]
I think that this is something related to react-live when you want to render something that needs to be transpiled before, what we need is transpile the code before rendering use something faster like bubble.

**Comment by jozefpetro** [comment]
Maybe similar problem here, if I'm using rest spread in playground I get an error:
```
Syntax Error: SyntaxError: This experimental syntax requires enabling the parser plugin: 'objectRestSpread'
```
Code:
```jsx
<Playground>
  <Component>
    {() => {
      const test = { a: 'a', b: 'b', c: 'c' }
      const testTwo = { a: 'a', b: 'b', c: 'c' }
      return (
        <pre>{JSON.stringify({ ...test, ...testTwo }, null, 2)}</pre>
      )
    }}
  </Component>
</Playground>
```

**Comment by pedronauck** [comment]
Closed in favor of #229
